File: datasets/IR-Plag/convertai-original-kotlin-rest-java/case-05/plagiarized/L4/01/L4.java
package IR;

public class L4 {

    public static void main(String[] args) 
    {
        int nomor = 0;
        System.out.print("Enter an integer: ");
        
        java.util.Scanner input = new java.util.Scanner(System.in);
        
        nomor = input.nextInt();
        
        int sisa = 0 ;   
        while (nomor != 0)      
        {
                sisa = nomor % 10;
                System.out.print(sisa);
                nomor = nomor / 10;
        }
        System.out.println();
    }

}

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-05/plagiarized/L4/01/L4.kt
package IR

fun main() {
    var nomor = 0
    println("Enter an integer: ")
    
    val input = java.util.Scanner(System.`in`)
    
    nomor = input.nextInt()
    
    var sisa = 0
    while (nomor != 0) {
        sisa = nomor % 10
        print(sisa)
        nomor /= 10
    }
    println()
}

